### PR TITLE
Feat: Add SBTMD Mobility Pass configuration

### LINKED ISSUE #1664: Create app configuration for SBMTD+Agency card (Mobility Pass)
## Tasks

- [x] Define `EligibilityType` for "SBMTD Mobility Pass"
- [x] Define `EligiblityVerifier` using MST CC as an example
  - [x] Need a per-verifier `SERVER_PUBLIC_KEY_URL` variable
- [x] Define necessary includes/templates
- [x] Define form `SBMTDMobilityPass` with configuration (labels, helper, validation, etc.)
- [x] Update SBMTD `TransitAgency`
- [x] Update all relevant copy for English and Spanish

## Acceptance criteria

- [x] SBMTD user sees an option for "Mobility Pass" on the Eligibility Index page
- [x] Selecting "Mobility Pass" shows more info on the Eligibility Start page
- [x] Continuing shows the Eligibility Confirm page with rendered form / details
- [x] English and Spanish match [`SBMTD` tab of spreadsheet](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit#gid=536435981)
- [x] Help page update
- [x] Submitting the form with sample data (e.g. against a local `server`) correctly verifies eligible and ineligible users

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #1664 

Note: Will require a PR into Eligibility Server first (for sample data and server.pub keys) before the form will return success for a eligible user correctly.

## Tasks

- [x] Define `EligibilityType` for "SBMTD Mobility Pass"
- [x] Define `EligiblityVerifier` using MST CC as an example
  - [x] Need a per-verifier `SERVER_PUBLIC_KEY_URL` variable
- [x] Define necessary includes/templates
- [x] Define form `SBMTDMobilityPass` with configuration (labels, helper, validation, etc.)
  - [x] Need new sample data
- [x] Update SBMTD `TransitAgency`
- [x] Update all relevant copy for English and Spanish

## Acceptance criteria

- [x] SBMTD user sees an option for "Mobility Pass" on the Eligibility Index page
- [x] Selecting "Mobility Pass" shows more info on the Eligibility Start page
- [x] Continuing shows the Eligibility Confirm page with rendered form / details
- [x] English and Spanish match [`SBMTD` tab of spreadsheet](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit#gid=536435981)
- [x] Help page update
- [x] Submitting the form with sample data (e.g. against a local `server`) correctly verifies eligible and ineligible users

### CODE REVIEW

File: benefits/core/migrations/0002_data.py [hunk 1]
```diff
@@ -28,10 +28,22 @@ def load_data(app, *args, **kwargs):
...
+        label="Eligibility server public key",
+        remote_url=os.environ.get(
+            "SERVER_PUBLIC_KEY_URL", "https://raw.githubusercontent.com/cal-itp/eligibility-server/dev/keys/server.pub"
+        ),
+    )
+
+    sbmtd_server_public_key = PemData.objects.create(
         label="Eligibility server public key",
         remote_url=os.environ.get(
             "SERVER_PUBLIC_KEY_URL", "https://raw.githubusercontent.com/cal-itp/eligibility-server/dev/keys/server.pub"
```
Comment (machikoyasuda) [inline benefits/core/migrations/0002_data.py:49]: Should these `SERVER_PUBLIC_KEY_URL` point somewhere else now? Create a separate `server.pub` files for SBMTD?

Comment (thekaveman) [reply benefits/core/migrations/0002_data.py:49]: Yeah I think that is what this task is trying to say:

> Need a per-verifier `SERVER_PUBLIC_KEY_URL` variable

In other words, we can't just have a single variable. We'd need `MST_SERVER_PUBLIC_KEY`, `SBMTD_SERVER_PUBLIC_KEY` etc.

**Comment by machikoyasuda** [comment]
@angela-tran @thekaveman I have this running locally with the latest `eligibility_server:dev`. I've confirmed the server has the new data. But I cannot get the app to successfully verify any of these new users: https://github.com/cal-itp/eligibility-server/pull/365/files#diff-ea8001f0f8d01218129545e693327a9226c0a2e8bbabf242441b44c2d6f0b204R18-R23

I'm not sure why.

**Comment by thekaveman** [comment]
@machikoyasuda I'm also failing to verify with the new users, but I noticed this in the logs of my local `eligibility-server` when I submitted the form:

```log
[2023-11-02 23:59:31,330] DEBUG eligibility_server.verify:104 List of types to check was empty.
```

So it looks like maybe the eligibility type isn't getting there?

File: benefits/core/migrations/0002_data.py [hunk 2]
```diff
@@ -337,9 +366,10 @@ def load_data(app, *args, **kwargs):
         index_template="core/index--sbmtd.html",
         eligibility_index_template="eligibility/index--sbmtd.html",
         enrollment_success_template="enrollment/success--sbmtd.html",
+        help_template="core/includes/help--sbmtd.html",
     )
     sbmtd_agency.eligibility_types.set([sbmtd_senior_type])
-    sbmtd_agency.eligibility_verifiers.set([sbmtd_senior_verifier])
+    sbmtd_agency.eligibility_verifiers.set([sbmtd_senior_verifier, sbmtd_mobility_pass_verifier])
```
Comment (thekaveman) [inline benefits/core/migrations/0002_data.py:372]: Oh interesting! I'm pretty sure this is the problem here:

`sbmtd_mobility_pass_verifier` was added to the agency as an eligibility verifier, but `sbmtd_mobility_pass_type` was not added to the agency's supported types.

But I'm surprised this didn't cause the Benefits app to crash, interesting... We should probably guard for this case where the `type` we are sending is empty, and throw an error.

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: benefits/core/migrations/0002_data.py (hunk 2 above)
Comment (machikoyasuda) [reply benefits/core/migrations/0002_data.py:372]: https://github.com/cal-itp/benefits/pull/1771/commits/e209a18fc2a53f81966933e0e7adb42a087cdd1a

**Comment by machikoyasuda** [comment]
Confirmed locally 
image
with credentials from https://github.com/cal-itp/eligibility-server/pull/365/files

**Comment by machikoyasuda** [comment]
Ready for re-review @thekaveman

**Comment by thekaveman** [comment]
@machikoyasuda will review today!

**Review by thekaveman — APPROVED** [review]
This looks good to me! I was able to run locally and verify a sample user.

Couple of things post-merge:

- [ ] I note that some of the language (particularly on the form) is slightly different from the spreadsheet; I agree with the version in this PR, since it matches more closely to Courtesy Cards. Please work with @indexing to make sure the spreadsheet is accurate.
- [ ] If you haven't already, please update the Key Vault secrets for the public keys: rename `server-public-key` to `mst-server-public-key` and add `sbmtd-server-public-key` for all 3 environments. Use a placeholder like `#` for SBMTD for now.

File: benefits/core/migrations/0002_data.py [hunk 3]
```diff
@@ -220,6 +232,23 @@ def load_data(app, *args, **kwargs):
         start_template="eligibility/start--senior.html",
     )
 
+    sbmtd_mobility_pass_verifier = EligibilityVerifier.objects.create(
+        name=os.environ.get("MOBILITY_PASS_VERIFIER_NAME", "Eligibility Server Verifier"),
+        active=os.environ.get("MOBILITY_PASS_VERIFIER_ACTIVE", "True").lower() == "true",
```
Comment (thekaveman) [inline benefits/core/migrations/0002_data.py:237]: Sorry just noticed something else.

We need an app config and key vault secrets for this `active` setting as well.

And let's go ahead and make the default `False` here so it doesn't turn on by accident.

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: benefits/core/migrations/0002_data.py (hunk 3 above)
Comment (thekaveman) [reply benefits/core/migrations/0002_data.py:237]: @machikoyasuda we'll need a follow-up PR for this.

Comment (thekaveman) [reply benefits/core/migrations/0002_data.py:237]: Actually, all the `MOBILITY_PASS_*` environment variables need key vault secrets defined in Terraform. I'll create a new issue.

**Pull request merged**